WebHostLib/templates/siteMap.html

```diff
@@ -26,6 +26,7 @@ <h2>Base Pages</h2>
             <li><a href="/user-content">User Content</a></li>
             <li><a href="/weighted-settings">Weighted Settings Page</a></li>
             <li><a href="{{url_for('stats')}}">Game Statistics</a></li>
+            <li><a href="/common-terms">Common Multiworld Terms</a></li>
```

Comment:
I almost used the word Jargon which would be more "correct" but I figured it might confuse users who don't know what Jargon means so opted to use "common terms" as not necessarily everything documented here is jargon